packages/ui/src/utils/steps.ts

```diff
@@ -1,5 +1,71 @@
 import type { Execute, RelayChain } from '@relayprotocol/relay-sdk'
-import type { Token } from '../types/index.js'
+import type { Token, LinkedWallet } from '../types/index.js'
+import { NormalizedWalletName } from '../constants/walletCompatibility.js'
+
+/**
+ * Get display name for wallet from linkedWallets array using current address
+ */
+const getWalletDisplayName = (
+  currentAddress?: string,
+  linkedWallets?: LinkedWallet[]
```

Comment:
Either we can pass in a defaultText which is used when the name is undefined or we just do the logic of using the default text outside of this function. It feels weird to return that by default, it could be that this is used elsewhere and that is not the intended default.